buckets.yaml

```diff
@@ -0,0 +1,17 @@
+# A mapping of bucket URLs to bucket metadata
+#
+# Possible bucket metadata:
+#   prefix: the identifier used for this bucket on the test-history dashboard.
+#   sequential: an optional boolean that indicates whether test runs in this
+#     bucket are numbered sequentially. This is used for an optimization in the
+#     test-history collection phase, and defaults to true.
+
+gs://kubernetes-jenkins/logs/:
+  prefix: ""
+gs://kube_azure_log/:
+  prefix: "azure$"
```

Comment:
I really dislike the $ and want to eliminate it for something that is easier on the eyes